package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -428,58 +745,102 @@ def __init__(self, universe, selection1='protein',
 
         If selection 1 and selection 2 are very mobile during the simulation
         and the contact surface is constantly changing (i.e. residues are
-        moving farther than 3 x `distance`), you might consider setting the
-        `update_selection1` and `update_selection2` keywords to ``True`` to
-        ensure correctness.
+        moving farther than 4 Å + `order` * (2 Å + `distance`)), you might
+        consider setting the `update_selection` keywords to ``True``
+        to ensure correctness.
```

Comment:
Add `.. versionchanged:: 0.20.0` note saying that `:attr:timeseries` output format changed and the class is now based on AnalysisBase.

Reply:
Note that you must have *two* blank lines between the `.. versionchanged` and the text above. It's a quirk in the sphinx.napoleon parser.

Reply:
The empty line and information have been added. I'm surprised that it looks fine when I use sphinx to render it.